tutorials/rise_mnist.ipynb

```diff
@@ -7,9 +7,9 @@
    "source": [
```

Comment:
<p>In the previous notebook you used 'relevance scores', which you are using in the rest of this notebook, too. Let's get rid of the 'attributions', even though a great term by itself, for consistency sake as to not confuse the user.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>